micro_sam/sam_annotator/image_series_annotator.py

```diff
@@ -180,17 +180,23 @@ def next_image(*args):
         nonlocal next_image_id
 
         segmentation = viewer.layers["committed_objects"].data
+        abort = False
         if segmentation.sum() == 0:
-            print("Nothing is segmented yet. Not advancing to next image.")
-            return
+            msg = "Nothing is segmented yet. Do you wish to continue to the next image?"
+            abort = widgets._generate_message("info", msg)
+            if abort:
+                return
 
         # Save the current segmentation.
         _save_segmentation(images[next_image_id], next_image_id, segmentation)
 
         # Load the next image.
         next_image_id += 1
         if next_image_id == len(images):
-            print("You have annotated the last image.")
+            msg = "You have annotated the last image."
+            print(msg)
+            # inform the user via dialog
+            # abort = widgets._generate_message("error", msg)
```

Comment:
I agree that we should open a dialog here. Let's discuss tomorrow what exactly we do in it.